logger/logger_writer.go

```diff
@@ -146,6 +147,8 @@ func queueMsg(lvl Level, prefix, format string, v ...interface{}) (err error) {
 		return
 	}
 
+	msg.time = time.Now()
+
```

Comment:
Not about this patch, but don't we need a msg.Reset before freeMsg(msg) at line 141
BTW, why is that defer declared inside the select?